src/pages/guidelines/ibm-dot-com-style-models/overview.mdx

```diff
@@ -0,0 +1,133 @@
+
+</AnchorLinks>
+
+
+## Our vision
+
+Our vision is that every visitor to IBM.com has a productive, efficient and delightful experience. IBM.com is a collection of highly functional, versatile, and diverse content and experiences. It serves different audience groups and their needs, in different industries and geographic regions, and covers a crucial portion of our users&rsquo; journeys.
+
+
+Here are the three principles that we set up to reach our vision of helping our users in the best way possible.
+
+<Title>1. The experience is the product</Title>
+
+The user&rsquo;s experience along the journey—spanning from the user&rsquo;s initial research through decision making, purchase, management, and all the way to the actual use of the given product—is becoming a core aspect of any product. 
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![User journeys](./images/journey-non-linear-dark.png)
+
+</Column>
+</Row>
+
+<Title>2. Drive consistency around user intents</Title>
+
+As more products are offered as services or components of larger ecosystems and platforms, users are increasingly traversing journey touch points in non-linear ways, which demands an even higher level of consistency and coherency across journeys.
+
+
+We drive consistency without sacrificing usability by basing the design of a page on the given user intent. For the same user intent, whether it&rsquo;s learning, trying, or buying, the solution should be consistent in order to accommodate the user&rsquo;s non-linear journey. This does not mean that everything should be the same—IBM.com style models define foundational and structural elements, while leaving the details to you, the maker.
+
+
+The ultimate end result is that the user should be instantly familiar with pages that solve similar problems.
+
+<Title>3. Reflect users&rsquo; journeys</Title>
+
+The design should reflect the progress the user is making towards the end goal of purchasing and using a product to solve their problems and achieve objectives. The style models were designed to show the progress the user is making.
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![Reflecting journey progression](./images/journey-progression-dark.png)
+
+</Column>
+</Row>
+
+
+## What are style models?
+A style model defines the foundational structure and elements of a given page to best serve the primary user intent of that page.
+
+
+Each style model is based on the user intent for a given page and defines the most optimal use of the foundational design elements—grid, color theme, type scale, spacing, layout, and screen regions—to ensure that the user is successful in achieving their goals (user intent).
+
+<Title>Primary user intent</Title>
+
+For any given page, there is an overarching user intent—we call this the primary user intent of a page. For example, learn is the primary user intent for an early learn search capture page even though the page mae have other secondary user intents such as navigation.
```

Comment:
Typo: change **mae** to **may**

Reply:
Thank you for catching this! I just committed a fix.